<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableWidget</class>
 <widget class="QWidget" name="TableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabWidgetPage1">
       <attribute name="title">
        <string>Data Sources</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTableView" name="dataSourceTableView"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="buttonSelectAll">
              <property name="text">
               <string>Select All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonDeselectAll">
              <property name="text">
               <string>Deselect All</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="buttonInvertSelection">
              <property name="text">
               <string>Invert Selection</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabWidgetPage2">
       <attribute name="title">
        <string>Settings &amp;&amp; Actions</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QGroupBox" name="groupBox_GeneralSettings">
            <property name="title">
             <string>Settings</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QCheckBox" name="checkBoxShowUnits">
               <property name="text">
                <string>Show Units in Table Headers</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="checkBoxOrientation">
               <property name="text">
                <string>Switch rows vs. columns</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_Coloring">
            <property name="title">
             <string>Use Values for Coloring</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_Coloring_AboveThreshold">
               <property name="text">
                <string>Color Above Threshold (&gt;)</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="button_Coloring_AboveThreshold">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_Coloring_Threshold">
               <property name="text">
                <string> Threshold</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBox_Coloring_Threshold">
               <property name="minimum">
                <double>-999999999.000000000000000</double>
               </property>
               <property name="maximum">
                <double>999999999.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_Coloring_BelowThreshold">
               <property name="text">
                <string>Color Below Threshold (&lt;=)</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QPushButton" name="button_Coloring_BelowThreshold">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_Actions">
            <property name="title">
             <string>Actions</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QPushButton" name="buttonSaveTable">
               <property name="text">
                <string>Save Data...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="buttonSetExperimentalData">
               <property name="text">
                <string>Add Data to Data Browser</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tableWrapper" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current plot.</string>
   </property>
  </action>
  <action name="actionAddToExperimentalData">
   <property name="text">
    <string>Add Data to Experimental Data</string>
   </property>
   <property name="toolTip">
    <string>Adds the currently displayed data and adds it to the experimental data. This is usually only useful if you want to want to use simulation results (which can then be perturbed) as input for other tools like parameter value estimation.</string>
   </property>
  </action>
  <action name="actionSelectAll">
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All Data Sources</string>
   </property>
  </action>
  <action name="actionDeselectAll">
   <property name="text">
    <string>Deselect All</string>
   </property>
   <property name="toolTip">
    <string>Deselect All Data Sources</string>
   </property>
  </action>
  <action name="actionInvertSelection">
   <property name="text">
    <string>Invert Selection</string>
   </property>
   <property name="toolTip">
    <string>Invert the Current Selection of Data Sources</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSetExperimentalData</sender>
   <signal>clicked()</signal>
   <receiver>actionAddToExperimentalData</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSelectAll</sender>
   <signal>clicked()</signal>
   <receiver>actionSelectAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDeselectAll</sender>
   <signal>clicked()</signal>
   <receiver>actionDeselectAll</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonInvertSelection</sender>
   <signal>clicked()</signal>
   <receiver>actionInvertSelection</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonSaveTable</sender>
   <signal>clicked()</signal>
   <receiver>actionSave</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>99</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
